src/deadline/maya_adaptor/MayaClient/render_handlers/arnold_handler.py

```diff
@@ -33,18 +33,42 @@ def start_render(self, data: dict) -> None:
 
         self.render_kwargs["camera"] = self.get_camera_to_render(data)
 
-        if "width" not in self.render_kwargs:
-            self.render_kwargs["width"] = maya.cmds.getAttr("defaultResolution.width")
-            print(
-                f"No width was specified, defaulting to {self.render_kwargs['width']}",
-                flush=True,
-            )
-        if "height" not in self.render_kwargs:
-            self.render_kwargs["height"] = maya.cmds.getAttr("defaultResolution.height")
-            print(
-                f"No height was specified, defaulting to {self.render_kwargs['height']}",
-                flush=True,
+        # In order of preference, use the task's output_file_prefix, the step's output_file_prefix, or the scene file setting.
+        output_file_prefix = data.get("output_file_prefix", self.output_file_prefix)
+        if output_file_prefix:
+            maya.cmds.setAttr(
+                "defaultRenderGlobals.imageFilePrefix", output_file_prefix, type="string"
             )
+            print(f"Set imageFilePrefix to {output_file_prefix}", flush=True)
+
+        if self.image_width is not None:
+            maya.cmds.setAttr("defaultResolution.width", self.image_width)
+            print(f"Set image width to {self.image_width}", flush=True)
+        if self.image_height is not None:
+            maya.cmds.setAttr("defaultResolution.height", self.image_height)
+            print(f"Set image height to {self.image_height}", flush=True)
+
+        region = [
```

Comment:
nit: I feel we would benefit by using a data class to model the region. The current implementation seems error prone. Did we choose this simpler approach because `region` is only used within these few lines of code?

Reply:
The current implementation is to keep things simple, yes. An advantage of this over a dataclass is that it has better cohesion compared to splitting it across a dataclass definition and where it is used.